cli/stork-publisher-agent/registry.go

```diff
@@ -0,0 +1,53 @@
+package stork_publisher_agent
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+)
+
+type RegistryClient struct {
+	baseUrl   string
+	authToken AuthToken
+}
+
+func NewRegistryClient(baseUrl string, authToken AuthToken) *RegistryClient {
+	return &RegistryClient{
+		baseUrl:   baseUrl,
+		authToken: authToken,
+	}
+}
+
+func (c *RegistryClient) GetBrokersForPublisher(publisherKey PublisherKey) (map[BrokerPublishUrl]map[AssetId]struct{}, error) {
+	brokerEndpoint := c.baseUrl + "/v1/registry/brokers"
+	queryParams := url.Values{}
+	queryParams.Add("publisher_key", string(publisherKey))
+	authHeader := http.Header{"Authorization": []string{"Basic " + string(c.authToken)}}
```

Comment:
this is kind of a cheat - we know that the auth token configured for use in the Stork Registry will also work to auth into the Stork Aggregator server.

Eventually this won't be the case, but we'll put the burden of authenticating public keys against the registry on the brokers, and we'll stop requiring auth tokens.